pkg/cmd/server/bootstrappolicy/infra_sa_policy.go

```diff
@@ -1122,4 +1129,57 @@ func init() {
 	if err != nil {
 		panic(err)
 	}
+
+	err = InfraSAs.addServiceAccount(
+		InfraTemplateInstanceControllerServiceAccountName,
+		authorizationapi.ClusterRole{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: TemplateInstanceControllerRoleName,
+			},
+			Rules: []authorizationapi.PolicyRule{
+				{
+					APIGroups: []string{templateapi.GroupName},
+					// "impersonate" is required for the API server to accept updates to
+					// TemplateInstance objects where the requester username is not the
+					// API caller.
+					Verbs:     sets.NewString("get", "list", "watch", "update", "impersonate"),
```

Comment:
I guess i need to refresh my memory on the flow.  I was assuming the broker created the templateinstance and the controller processed it, but it looks like you've got a brokertemplateinstance resource *and* a templateinstance resource.